Mathlib/Topology/ContinuousMap/Interval.lean

```diff
@@ -0,0 +1,148 @@
+/-
+Copyright (c) 2024 Vincent Beffara. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Vincent Beffara
+-/
+import Mathlib.Topology.CompactOpen
+import Mathlib.Topology.Order.ProjIcc
+
+/-!
+# Continuous bundled maps on intervals
+
+In this file we prove a few results about `ContinuousMap` when the domain is an interval.
+-/
+
+open Set ContinuousMap Filter Topology
+
+namespace ContinuousMap
+
+variable {α : Type*} [LinearOrder α] [TopologicalSpace α] [OrderTopology α]
+variable {a b c : α} [Fact (a ≤ b)] [Fact (b ≤ c)]
+variable {E : Type*} [TopologicalSpace E]
+
+/-- The embedding into an interval from a sub-interval lying on the left, as a `ContinuousMap`. -/
+def subintervalLeft : C(Icc a b, Icc a c) :=
+  .inclusion <| Icc_subset_Icc le_rfl Fact.out
```

Comment:
Are you sure no one will ever need the other `Ixx` versions? I wonder whether `Icc` should appear anywhere in the name

Reply:
`IccInclusionLeft` then?